kolibri/plugins/management/assets/src/views/class-enroll-page/index.vue

```diff
@@ -324,24 +346,21 @@
 
   @require '~kolibri.styles.definitions'
 
-  $toolbar-height = 36px
+  $table-row-selected = #e0e0e0
+  $table-row-hover = #eee
```

Comment:
If they are not used anywhere else, best to keep them in the `<style></style>` block of the component for now - but it would be nice to use the same colours across the app for selected and hovered rows, certainly!